README.md

```diff
@@ -21,36 +25,53 @@ var sharetools = new ShareTools({
             subject: 'SUPER IMPORTANT EMAIL',
             message: 'BBC News has new bespoke'
         },
-        app: {
-            title: 'Shiny new app',
-            text: 'Hello world'
+        // you can also specify custom social networks, e.g.
+        whatsapp123: {
```

Comment:
We could split out the initialisation of custom networks from the initialisation of data.

So something like
```
sharetools.addNetworkConfigs(ifsNetworkConfigs)
```

That way we can keep all the extra network configs in a separate file which we require in.